adapters/restclient/lib/opentelemetry/adapters/restclient/adapter.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Adapters
+    module RestClient
+      # The Adapter class contains logic to detect and install the RestClient
+      # instrumentation adapter
+      class Adapter < OpenTelemetry::Instrumentation::Adapter
+        install do |config|
+          require_dependencies
+          config[:request_patch] ||= Patches::Request
```

Comment:
Is there any reason to make the request patch configurable at this point?

Reply:
https://github.com/open-telemetry/opentelemetry-ruby/pull/176#discussion_r374187096

Not that I know of, but I wanted to verify with you before removing.